server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/CompactableUtils.java

```diff
@@ -383,50 +307,33 @@ public Optional<SortedKeyValueIterator<Key,Value>> getSample(CompactableFile fil
 
   private static final class TableCompactionHelper implements CompactionHelper {
     private final PluginConfig cselCfg2;
-    private final CompactionStrategyConfig stratCfg2;
     private final Tablet tablet;
-    private WriteParameters wp;
-    private Set<StoredTabletFile> filesToDrop;
 
-    private TableCompactionHelper(PluginConfig cselCfg2, CompactionStrategyConfig stratCfg2,
-        Tablet tablet) {
-      this.cselCfg2 = cselCfg2;
-      this.stratCfg2 = stratCfg2;
-      this.tablet = tablet;
+    private TableCompactionHelper(PluginConfig cselCfg2, Tablet tablet) {
+      this.cselCfg2 = Objects.requireNonNull(cselCfg2);
+      this.tablet = Objects.requireNonNull(tablet);
     }
 
     @Override
     public Set<StoredTabletFile> selectFiles(SortedMap<StoredTabletFile,DataFileValue> allFiles) {
-      if (cselCfg2 != null) {
-        filesToDrop = Set.of();
-        return CompactableUtils.selectFiles(tablet, allFiles, cselCfg2);
-      } else {
-        var plan =
-            CompactableUtils.selectFiles(CompactionKind.SELECTOR, tablet, allFiles, stratCfg2);
-        this.wp = plan.writeParameters;
-        filesToDrop = Set.copyOf(plan.deleteFiles);
-        return Set.copyOf(plan.inputFiles);
-      }
+      return CompactableUtils.selectFiles(tablet, allFiles, cselCfg2);
     }
 
     @Override
     public Map<String,String> getConfigOverrides(Set<CompactableFile> files) {
-      return computeOverrides(wp);
+      return null;
     }
 
     @Override
     public Set<StoredTabletFile> getFilesToDrop() {
```

Comment:
This method should be completely removed from the code as this functionality of dropping files was unique to the compaction strategy and is not supported by the compaction selector.

Reply:
`getFilesToDrop()` has been removed as well as places that use it